models/base/scratch/bigquery/snowplow_mobile_base_events_this_run.sql

```diff
@@ -66,13 +61,13 @@ with events as (
   inner join {{ ref('snowplow_mobile_base_sessions_this_run') }} as b
   on {{ session_id }} = b.session_id
 
-  where {{ snowplow_utils.timestamp_diff('a.collector_tstamp', 'b.start_tstamp', 'day')}} <= {{ var("snowplow__max_session_days", 3) }}
-  and {{ snowplow_utils.timestamp_diff('a.dvce_created_tstamp', 'a.dvce_sent_tstamp', 'day') }} <= {{ var("snowplow__days_late_allowed", 3) }}
+  where {{ snowplow_utils.timestamp_add('day', var("snowplow__max_session_days", 3), 'a.collector_tstamp') }} <= b.start_tstamp
```

Comment:
I think this is the wrong way around and will also not allow for partition pruning. I think we can copy the logic from the web model